deployments/charts/osmo/README.md

```
@@ -90,26 +92,22 @@ helm --kube-context kind-osmo upgrade --install osmo deployments/charts/osmo \
 
 The first installation uses the MEK bootstrap lifecycle Job to create the
 retained `osmo-master-encryption-key` Secret without putting key material in
-Helm state. After that installation succeeds, remove the temporary Secret
-creation permission and retain the remaining release values:
+Helm state. Bootstrap can remain enabled for the Quickstart. It must be
+disabled before rotating the MEK, as described in
+[Master encryption key lifecycle](#master-encryption-key-lifecycle).
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)